src/httprpc.cpp

```diff
@@ -184,6 +191,23 @@ static bool HTTPReq_JSONRPC(HTTPRequest* req, const std::string &)
         // Set the URI
         jreq.URI = req->GetURI();
 
+#ifdef ENABLE_WALLET
+        if (walletName.empty()) {
```

Comment:
Agree with @ryanofsky. There has been an effort to avoid more wallet dependencies.